Library/Formula/iojs.rb

```diff
@@ -0,0 +1,39 @@
+class Iojs < Formula
+  homepage "https://iojs.org/"
+  url "https://iojs.org/dist/v1.0.4/iojs-v1.0.4.tar.xz"
+  version "1.0.4"
+  sha1 "6247d3bb544353cf134e0857f4a60e7e04959bd4"
```

Comment:
sha256 would be great here, since that's what ` iojs ` publish as their checksums I believe?